packages/image_picker/image_picker_ios/lib/image_picker_ios.dart

```diff
@@ -2,12 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:async';
-
+import 'package:image_picker_ios/src/messages.g.dart';
 import 'package:image_picker_platform_interface/image_picker_platform_interface.dart';
 
-import 'src/messages.g.dart';
```

Comment:
Why did you change this to a package-style import within the same package?

Reply:
I was checking if it could fix an issue with imports related to Pigeon, but then I've found a GitHub issue about it and forgot to revert this change. I'll do it now.